test/built-ins/Proxy/has/return-true-without-same-target-prop.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.7
+description: >
+    The result of [[HasProperty]] is a Boolean value and will affect has
+    checkings. True returned when target property doesn't exists;
+---*/
```

Comment:
you mean `[noStrict]`, right? As with doesn't work on strict mode and by the other examples I found, I'm using this flag now.